changelog/b11997.dd

```diff
@@ -0,0 +1,14 @@
+rdmd now checks for a D compiler binary in the directory
+it's in first
```

Comment:
First changelog line has to be all on its own line (not wrapped)... CI / changelog generation will fail otherwise.